h2o-docs/src/product/data-science/algo-params/max_iterations.rst

```diff
@@ -1,7 +1,7 @@
 ``max_iterations``
 ------------------
 
-- Available in: GLM,  GAM, PCA, GLRM, K-Means, CoxPH
+- Available in: GLM, GAM, HGLM, PCA, GLRM, K-Means, CoxPH
```

Comment:
Please add the following algos to the end of AdaBoost:

ANOVAGLM, ModelSelection